xl/common.py

```diff
@@ -294,9 +308,9 @@ def wrapper(*args, **kwargs):
         stats = pstats.Stats(prof)
         stats.strip_dirs()
         stats.sort_stats('time', 'calls')
-        print ">>>---- Begin profiling print"
+        print(">>>---- Begin profiling print")
```

Comment:
It is only needed for pythons < 2.6 if I understand correctly. 
And yes, it is required to use .format() to print multiple values the same way in python3 and python2.